PeerioTechnologies/peerio-mobile
# feat: redesign of file view, inline file download

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/302/mobile-new-file-details-view-and-unable-to-cancel-downloads
https://github.com/PeerioTechnologies/peerio-icebear/pull/181

#### Testing instructions  
Files view
* new menu icon to the right of each file: check that all actions are working

File details view (from Files view, tap on an individual file)
* completely redesigned
* check that download/cancel/open actions work in the main view
* new menu icon in top right; check that all actions are working

Inline files in chat (non-image files only)
* check that new download/cancel/open actions work in the new grey button below the file name
* some modifications to the menu icon in the right; check that menu actions are working

**Note**: in all cases, the "open" action attempts to open the file in the device's image viewer, even if the file is not an image. This was true for me on the emulator, as well as on my phone (Android) using prod. Karim reproduced on his Android phone. If this behaviour can be confirmed, then we should log it but as a separate bug (out of scope for this feature).

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
